python/cudf/cudf/core/udf/pipeline.py

```diff
@@ -1,28 +1,42 @@
+import cachetools
+import numpy as np
+from numba import cuda
 from numba.np import numpy_support
+from numba.types import Tuple, boolean, int64, void
 from nvtx import annotate
 
+from cudf.core.udf.api import Masked, pack_return
 from cudf.core.udf.typing import MaskedType
 from cudf.utils import cudautils
 
+libcudf_bitmask_type = numpy_support.from_dtype(np.dtype("int32"))
+MASK_BITSIZE = np.dtype("int32").itemsize * 8
+precompiled: cachetools.LRUCache = cachetools.LRUCache(maxsize=32)
+
+cuda.jit(device=True)(pack_return)
```

Comment:
I believe this is superfluous - since you also implement typing and lowering for `pack_return`, it doesn't need to be jitted (and if this jitted version ever got used, it would not do anything because it jits a function with an empty body).